lib/Features/FeatureFlagsRepository.php

```diff
@@ -15,4 +15,33 @@ class FeatureFlagsRepository extends Repository {
   protected function getEntityClassName() {
     return FeatureFlagEntity::class;
   }
+
+  /**
+   * @param array $data
+   * @throws \RuntimeException
+   * @throws \InvalidArgumentException
+   * @return FeatureFlagEntity
+   */
+  public function createOrUpdate(array $data = []) {
+    if (!$data['name']) {
+      throw new \InvalidArgumentException('Missing name');
+    }
+    $feature_flag = $this->findOneBy([
+      'name' => $data['name'],
+    ]);
+    if (!$feature_flag) {
+      $feature_flag = new FeatureFlagEntity($data['name']);
+      $this->persist($feature_flag);
+    }
+    if (isset($data['value'])) {
+      $feature_flag->setValue($data['value']);
+    }
+
+    try {
+      $this->flush();
+    } catch (\Exception $e) {
+      throw new \RuntimeException("Error when saving feature " . $data['name']);
+    }
```

Comment:
Not sure if we need to rethrow this exception. Maybe the try/catch is not necessary. Failing to save to DB would definitely be a severe error that we don't need to handle I guess. Or does this try to prevent exposing some data, etc.? My point is - we'll probably use `flush()` in many other places and we won't wrap them there I guess.

Reply:
I tried to leave the functionality as it was before. The previous code was throwing this exception if saving failed, so I left it there.